app/templates/services/service_submission.html

```diff
@@ -187,7 +187,7 @@ <h2 class="temporary-message-heading" id="temporary-message-heading">
 
 {% block after_sections %}
   {% if not delete_requested %}
-    <div class="grid-row">
```

Comment:
why is this being deleted? if we using a column class then it should be inside a grid row. 🤔 not sure if it might be worth renaming the classes here to use govuk-frontend classes but I'm happy if you decide to leave it for another ticket

Reply:
It already is inside a grid row! I spotted some weird alignment when QAing locally and found it was extending `grid-row` from `_base_service_page.html`. (I should have mentioned this in the PR description, whoops.)